helm/minio/templates/statefulset.yaml

```diff
@@ -87,6 +87,7 @@ spec:
         runAsUser: {{ .Values.securityContext.runAsUser }}
         runAsGroup: {{ .Values.securityContext.runAsGroup }}
         fsGroup: {{ .Values.securityContext.fsGroup }}
+        fsGroupChangePolicy: {{ .Values.securityContext.fsGroupChangePolicy }}
```

Comment:
this is only available in v1.20 k8s so this should be optional otherwise this will fail under older k8s @dharapvj

Reply:
aah .. good point.. Checking how to put that check in helm chart!

Reply:
Check now.. if the additional check will suffice.